packages/js/product-editor/src/hooks/use-product-manager/use-product-manager.ts

```diff
@@ -70,17 +71,21 @@ export function useProductManager< T = Product >( postType: string ) {
 			setIsSaving( true );
 
 			await validate( extraProps );
+			const { saveEntityRecord } = dispatch( 'core' );
 
-			// @ts-expect-error There are no types for this.
-			const { editEntityRecord, saveEditedEntityRecord } =
-				dispatch( 'core' );
-
-			await editEntityRecord< T >( 'postType', postType, id, extraProps );
+			const { blocks, content, selection, ...editedProduct } = WPSelect(
```

Comment:
We extract the props `blocks`, `content`, and `selection` because they are not really part of the product, and that would create unexpected side effects